stdpopsim/catalog/BosTau/species.py

```diff
@@ -67,39 +73,8 @@
 
 # Generic and chromosome-specific ploidy
 _species_ploidy = 2
-_ploidy = {
-    "1": _species_ploidy,
-    "2": _species_ploidy,
-    "3": _species_ploidy,
-    "4": _species_ploidy,
-    "5": _species_ploidy,
-    "6": _species_ploidy,
-    "7": _species_ploidy,
-    "8": _species_ploidy,
-    "9": _species_ploidy,
-    "10": _species_ploidy,
-    "11": _species_ploidy,
-    "12": _species_ploidy,
-    "13": _species_ploidy,
-    "14": _species_ploidy,
-    "15": _species_ploidy,
-    "16": _species_ploidy,
-    "17": _species_ploidy,
-    "18": _species_ploidy,
-    "19": _species_ploidy,
-    "20": _species_ploidy,
-    "21": _species_ploidy,
-    "22": _species_ploidy,
-    "23": _species_ploidy,
-    "24": _species_ploidy,
-    "25": _species_ploidy,
-    "26": _species_ploidy,
-    "27": _species_ploidy,
-    "28": _species_ploidy,
-    "29": _species_ploidy,
-    "X": _species_ploidy,
-    "MT": 1,
-}
+_ploidy = {str(i): _species_ploidy for i in range(1, 30)}
+_ploidy.update({"X": _species_ploidy, "MT": 1})
 
 _chromosomes = []
```

Comment:
`_chromosomes` is no longer used

Reply:
yes that's right. forgot to delete these lines. will do